\documentclass[11pt,a4paper,twoside]{epig}
\input{epig-macros}

\hyphenpenalty 1000

\newcommand{\FIG}[4]
{\begin{figure}[!hbt]
\begin{center}
\rotatebox{270}{\includegraphics[width=#1]{#2}}
\caption{\label{fig:#3}#4\vspace{-5mm}}
\end{center}
\end{figure}}

\begin{document}

\titre{Hypothesis: Variations in the rate of DNA replication determine the phenotype of daughter cells}
\auteur{Vic Norris$^{1,4}$, Laurent Jannière$^2$ and Patrick Amar$^{3,4}$}
\affiliation{$^1 $
UMR CNRS 6522, Department of Science, Univ. of Rouen, F-76821, France}
\\
\affiliation{$^2 $
Laboratoire de Génétique Microbienne, INRA, F-78352 Jouy en Josas, France}
\\
\affiliation{$^3 $
LRI, Univervité Paris-Sud \& UMR CNRS 8623, F-91405 Orsay, France}
\\
\affiliation{$^4 $
Epigenomics Project, Genopole\textsuperscript{\textregistered}, F-91000 Evry, France}

\section*{Abstract}
\vspace{-2mm}
The existence of two identical chromosomes within the same cell in which genes and higher order
structures compete for limited resources is a symmetry-breaking situation previously proposed to
lead to differentiation. Recent experiments are consistent with an intimate relationship between
metabolism and the rate of chromosome replication in bacteria. The process of chromosome
replication progressively changes the copy number of genes and sites in a linear order. This raises
the possibility that slowing or even pausing replication for different times at different sites in
the chromosome might be combined with various mechanisms leading to local cooperation and global
competition. If so, such replication-phenotype coupling could produce different patterns of gene
expression. Indeed, replication-phenotype coupling may constitute a powerful and fundamental way of
generating coherent phenotypes. As a prelude to testing this hypothesis, we discuss some of the
parameters that will need to be explored by bench experimentation and computer simulation.

\vspace{-2mm}
\section{Introduction}

One of the fundamental problems in biology, highlighted by Kauffman \cite{14}, is how cells
integrate gene expression and environmental conditions to steer their phenotypes in a coherent,
reproducible way through the vast space of possibilities apparently available to them. A possible
solution is that the very existence of two chemically identical chromosomes in the same cytoplasm
spontaneously leads to different patterns of gene expression and that this underpins
differentiation \cite{25}. This is based on the idea that if a gene attracts an RNA polymerase it
has a greater chance of attracting a second one and hence, if two identical copies of a gene
compete for a limited number of RNA polymerases, one copy is expressed and the other silent.
Related ideas about the primordial role of the cell cycle in generating not just diversity but
coherent diversity have also been developed \cite{30,24}.

\vspace{3mm}

Such ideas need to be updated in the context in which gene expression takes place at the level
of hyperstructures which are large, spatially extended assemblies of ions, molecules and
macromolecules that are implicated in functions that range from DNA replication and cell
division to chemotaxis and secretion \cite{23}. These ideas also require updating due to the
discovery that carbon metabolism in \emph{Bacillus subtilis}, and almost certainly other
bacteria, affects the enzymes responsible for the elongation step in chromosome replication
\cite{39}. In other words, metabolism appears to be exerting a direct control over the
way the chromosome is replicated. This suggests to us a reciprocal relationship in which the
way the chromosome is replicated determines the phenotype. Here we explore this idea.

\vspace{3mm}
\noindent
...


\begin{thebibliography}{10}

\bibitem{39} Jannière L., D. Canceill, C. Suski, S. Kanga, B. Dalmais, R. Lestini, A.-F. Monnier, J. Chapuis, A. Bolotin,  M. Titok, E. Le Chatelier and S. D. Ehrlich, (2007) Genetic Evidence for a Link Between Glycolysis and DNA Replication. \emph{PLoS ONE} \textbf{5}: e447.
\bibitem{14} Kauffman, S., (1993) The origins of order. \emph{Oxford University Press}, Oxford.
\bibitem{23} Norris, V., A. Cabin-Flaman, T. den Blaauwen, R. H. Doi, R. Harshey, L. Jannière, A. Jimenez- Sanchez, D. J. Jin, P. A. Levin, E. Mileykovskaya, A. Minsky, M. Saier Jnr. \& K. Skarstad, (2007) Functional Taxonomy of Bacterial Hyperstructures. \emph{Microbiology and Molecular Biology} \textbf{71}:~230-253.
\bibitem{24} Norris, V., M. Demarty, D. Raine, A. Cabin-Flaman \& L. Le Sceller, (2002) Hypothesis: hyperstructures regulate initiation in Escherichia coli and other bacteria. \emph{Biochimie} \textbf{84}: 341-347.
\bibitem{25} Norris, V. \& M. S. Madsen, (1995) Autocatalytic gene expression occurs via transertion and membrane domain formation and underlies differentiation in bacteria: a model. \emph{Journal of molecular biology} \textbf{253}:~739-748.
\bibitem{30} Segre, D., D. Ben-Eli \& D. Lancet, (2000) Compositional genomes: prebiotic information transfer in mutually catalytic noncovalent assemblies. \emph{Proceedings of the National Academy of Science U.S.A.} \textbf{97}: 4112-4117.

\end{thebibliography}
\end{document}
